docs/victoriametrics/README.md

```diff
@@ -483,6 +483,7 @@ To get metric names usage statistics, use the `/prometheus/api/v1/status/metric_
 * `limit` - integer value to limit the number of metric names in response. By default, API returns 1000 records.
 * `le` -  `less than or equal`, is an integer threshold for filtering metric names by their usage count in queries. For example, with `?le=1` API returns metric names that were queried <=1 times.
 * `match_pattern` - a substring pattern to match metric names. For example, `?match_pattern=vm_` will match any metric names with `vm_` pattern, like `vm_http_requests`, `max_vm_memory_available`. It doesn't support regex syntax.
+* `match_names` - a list of metric names to query. If this param is defined, `le` and `match_pattern` options will be ignored.
```

Comment:
Could you please add more details on this: example, separating multiple names, escaping special chars?

Reply:
what is the filtering order:
1. is match_names applied first or match_pattern?
2. are match_names and match_pattern exclusive?
3. when limit is applied: before or after?

Reply:
@f41gh7 what would the performance impact of `match_pattern ` to support regex?
 If impact is relateively low on theoretical load of 10k metric names, I would prefer it to support regex rather than introducing a new GET param.

Reply:
match_pattern regex requires full scan. It'll work slowly with time and may have impact on data ingestion due to lock read lock on tracker.

 For the most use cases match for exact metric names list will be fast and efficient.

Reply:
That's unfrotunate. From user PoV match_names and  match_pattern are conflicting params. 
But if there is no another way, then this is the way.

Reply:
Yes, I also do not like that. However, I think. It's better to add regex support from usability perspective. Requests to this API shouldn't be frequent.